packages/theme/src/types.ts

```diff
@@ -43,7 +43,10 @@ type PaletteAdditions = {
   linkHover: string;
   link: string;
   gold: string;
-  sidebar: string;
+  sidebar: {
+    backgroundColor: string;
+    selectedIndicatorColor: string;
```

Comment:
Probably just `indicator` here imo, like the tabbar.

Reply:
Agreed, updated.